src/MicrosoftTeams.schema.json

```diff
@@ -173,36 +172,107 @@
                         "description": "A unique identifier for the bot that matches its ID in the bot framework.",
                         "maxLength": 64
                     },
+                    "needsChannelSelctor": {
+                        "type": "boolean",
+                        "description": "This value describes whether or not the bot utilizes a user hint to add the bot to a specific channel.",
+                        "default": "false"
+                    },
                     "scope": {
                         "type": "array",
+                        "description": "Specifies whether the bot offers an experience in the context of a channel in a team, or an experience scoped to an individual user alone. These options are non-exclusive",
                         "items": {
                             "enum": [
                                 "team",
-                                "people",
                                 "personal"
-                            ],
-                        "description": "Specifies whether the bot offers an experience in the context of a channel in a team, a chat between one or more people, or an experience scoped to an individual user alone. These options are non-exclusive"
+                            ]
+                        }
+                    },
+                    "commandList": {
+                        "type": "array",
```

Comment:
Since we went with distinct commandLists per scope I don't think we should be trying to merge commands across different menus. In other words, you can have no command list, a command list per scope, or a unified command list that works in all scopes. In any case, the maximum count here isn't a big deal; what's more important is that we document what we expect here from developers.